simnode/generator.py

```diff
@@ -235,8 +281,8 @@ def get_obstacle_common(node: _Element) -> Tuple[float, float, float, float, flo
         :param node: The obstacle node
         :return: x, y, x_rot, y_rot, z_rot, height, id
         """
-        return float(node.get("x")), float(node.get("y")), float(node.get("x_rot", 0)), float(node.get("y_rot", 0)), \
-               float(node.get("z_rot", 0)), float(node.get("height")), node.get("id", None)
+        return float(node.get("x")), float(node.get("y")), float(node.get("xRot", 0)), float(node.get("yRot", 0)), \
```

Comment:
x_rot and others need to be replaced with xRot, otherwise rotations aren't recognized